test/dsets.c

```diff
@@ -15558,11 +15558,10 @@ main(void)
     int         nerrors = 0;
     const char *envval;
     hbool_t     contig_addr_vfd; /* Whether VFD used has a contigous address space */
-    hbool_t     driver_uses_modified_filename = h5_driver_uses_modified_filename();
     int         i;
 
     /* Don't run this test using certain file drivers */
-    envval = HDgetenv(HDF5_DRIVER);
+    envval = HDgetenv("HDF5_DRIVER");
```

Comment:
This was intentionally changed to be a macro and not a string so that all occurrences get updated if the env. variable name is ever changed.